PC/hu_moments_gather/hu_moments_gather.cpp

```diff
@@ -0,0 +1,106 @@
+#include <iostream>
+#include <vector>
+
+#include <opencv2/highgui/highgui.hpp>
+#include "opencv2/imgproc/imgproc.hpp"
+
+#include "print_vector.h"
+
+#define SHOW_IMAGES     1
```

Comment:
maybe CLI parameter instead of #define ? (Needs no recompilation)

Reply:
Do you know a nice way to have named CLI arguments to C++ programs?